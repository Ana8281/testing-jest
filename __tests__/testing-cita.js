import Citas from "../js/classes/Citas";

describe('Citas', () => {
    const citas = new Citas();

    const id = Date.now();

    test('Add Citas', () => {
        const citaObject = {
            id,
            fecha: "03.05.2020",
            hora: "10:30",
            mascota: "Sam",
            propietario: "Laura",
            sintomas: "Fiber",
            telefono: "333",
        }

        citaObject.id = id;

        citas.agregarCita(citaObject)

        expect(citas).toMatchSnapshot()
    })

    test('update Citas', () => {
        const citaUpdated = {
            id,
            fecha: "03.05.2020",
            hora: "10:30",
            mascota: "Simon",
            propietario: "Laura",
            sintomas: "Fiber",
            telefono: "333",
        }


        citas.editarCita(citaUpdated)

        expect(citas).toMatchSnapshot()
    })


    test('delete Citas', () => {
        citas.eliminarCita(id)

        expect(citas).toMatchSnapshot()
    })
})