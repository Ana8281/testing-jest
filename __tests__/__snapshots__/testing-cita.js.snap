// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Citas Add Citas 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "03.05.2020",
      "hora": "10:30",
      "id": 1624347471218,
      "mascota": "Sam",
      "propietario": "Laura",
      "sintomas": "Fiber",
      "telefono": "333",
    },
  ],
}
`;

exports[`Citas delete Citas 1`] = `
Citas {
  "citas": Array [],
}
`;

exports[`Citas update Citas 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "03.05.2020",
      "hora": "10:30",
      "id": 1624347471218,
      "mascota": "Simon",
      "propietario": "Laura",
      "sintomas": "Fiber",
      "telefono": "333",
    },
  ],
}
`;
